app/static/src/scss/includes/_static-content-template.scss

```diff
@@ -1,9 +1,15 @@
-/* SCSS for How-to-use-this-service page */
+/* SCSS for static content template */
 
 .govuk-grid-column-one-third {
   &--ayr-content {
     position: sticky;
+    max-height: 100dvh;
+    display: flex;
+    flex-direction: column;
     top: 1rem;
+    @media screen and (width <=810px) {
```

Comment:
so when width is less than or equal to 810px we make position of this static, not sticky as per the AA report suggestion but whats the use of lines 6-8?

Reply:
I don't think we want the box to be sticky unless the columns are side by side like on larger devices.

We shouldn't have sticky content on top of scrollable content else that is a WCAG 2.2 2.4.11 failure (https://www.w3.org/WAI/WCAG22/Understanding/focus-not-obscured-minimum.html).

If the window is not as high/tall as the sidebar then you can never get to the last two links until you scroll to the bottom of the "sticky patch".

The lines _should_ mean you can scroll within the sidebar separately to the page.

Reply:
okay, thanks for clarifying 👌